local-development.mdx

```diff
@@ -198,5 +183,5 @@ During development you can easily connect to a SQLite, libSQL, or Turso database
 - [Outerbase](https://www.outerbase.com) &mdash; Runs in the browser
 - [TablePlus](https://tableplus.com) &mdash; macOS, Windows, and Linux
 - [Dataflare](https://dataflare.app) &mdash; Paid (with limited free version) macOS, Windows, and Linux
-- [libSQL Studio](https://github.com/invisal/libsql-studio) - Runs in the browser
+- [Outerbase Studio](https://libsqlstudio.com) - Runs in the browser
```

Comment:
unrelated?

Reply:
Unrelated to extensions yeah, but decided to apply some stashed changes inside this PR as well.